gradle-enterprise-conventions-gradle-plugin/build.gradle

```diff
@@ -18,7 +18,7 @@ configurations {
 
 dependencies {
 	implementation(project(":gradle-enterprise-conventions-core"))
-	compatibilityTestImplementation("com.gradle:gradle-enterprise-gradle-plugin:3.1.1")
+	compatibilityTestImplementation("com.gradle:gradle-enterprise-gradle-plugin:3.4")
```

Comment:
@runningcode shouldn't this be 3.4.1, too?

Reply:
I figured we would want to test compatibility against the oldest possible version.
The previous logic was testing against 3.1.1 while running 3.3.3.

Reply:
Got it.